src/Titanium.Web.Proxy/Compression/CompressionFactory.cs

```diff
@@ -18,6 +19,13 @@ internal static Stream Create(string type, Stream stream, bool leaveOpen = true)
                     return new GZipStream(stream, CompressionMode.Compress, leaveOpen);
                 case KnownHeaders.ContentEncodingDeflate:
                     return new DeflateStream(stream, CompressionMode.Compress, leaveOpen);
+                case KnownHeaders.ContentEncodingBrotli:
+                    if (!RunTime.IsWindows)
```

Comment:
Why this is needed. See my comment on #494

Reply:
Removed in latest commit. I forgot that this wasn't a native+P/invoke solution.